app/src/config.js

```diff
@@ -74,6 +74,8 @@ export const TIMELINE_OVERALL_START_DATE_OFFSET = Math.floor(TIMELINE_OVERALL_ST
 // ie how large the clicked region should be for including vessels
 export const VESSEL_CLICK_TOLERANCE_PX = 3.5;
 
+export const AIS_ID = 'fishing2';
```

Comment:
you-in-6-months will have no clue what "AIS_ID" means or is used for. Don't be afraid to use long, descriptive variable names. please improve